scripts/lib/pr-body-contract.mjs

```
@@ -2,6 +2,93 @@
 
 import { readFileSync } from 'node:fs';
 
+// HTML block starters shared between the top-level block dispatch below and
+// interruptsParagraph()'s CommonMark paragraph-interruption check, so the
+// two classifications cannot drift apart.
+// GFM/CommonMark's inter-token whitespace is space/tab only, not the full
+// Unicode set JavaScript's \s matches (e.g. U+2003). Using \s here let a
+// Unicode-whitespace-separated tag like "<pre >" open a raw HTML block
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)